src/components/DeckMechanisms/index.js

```diff
@@ -156,8 +167,23 @@ export default class DeckMechanisms extends React.Component {
           case 'W2':
           case 'W6':
           case 'W11':
-            // Frosthexers, Moment's Peace, Midwinter Chaos
-            newState.specifics.potentialFrozenEnemies = true
+          case 'W4':
+            const frozenEnemiesNowRegular =
```

Comment:
```suggestion
            const unfrozenEnemies =
```

Reply:
This variable indicates how many frozen enemies left, not how many were unfrozen/destroyed by Wisp Cloud. I would keep this as it is

Reply:
I don’t fully get the “nowRegular”. What does that mean in that context?

Reply:
now means once the ability has been resolved, regular indicates the RNG mode